### DISCUSSION TITLE
Add state for 'overflow: ...' so you can conditionally add classes when content overflows

### DISCUSSION THREAD

**Discussion by sp88011** [tailwindlabs/tailwindcss#10689] [Ideas]
Have a state that indicates whether content overflows. 

E.g. `overflow-y:pr-3` would add padding to the right whenenver the content overflows on the y-axis.

**Comment by luizov** [comment]
CSS alone is not able to check a condition for example if a box is overflowing. A javascript solution has to be implemented for this.
